ui/pages/confirmations/components/confirm/info/token-transfer/transaction-flow-section.tsx

```diff
@@ -50,24 +56,40 @@ export const TransactionFlowSection = () => {
         flexDirection={FlexDirection.Row}
         justifyContent={JustifyContent.spaceBetween}
         alignItems={AlignItems.center}
-        padding={3}
       >
-        <Name
-          value={transactionMeta.txParams.from}
-          type={NameType.ETHEREUM_ADDRESS}
-          variation={chainId}
-        />
+        <ConfirmInfoAlertRow
+          alertKey={RowAlertKey.SigningInWith}
+          label={t('from')}
+          ownerId={transactionMeta.id}
+        >
+          <Box marginTop={1}>
```

Comment:
Frustrating we can't use `SigningInWithRow` here but it seems the `label` and this `Box` are unique.